test/e2e/tests/collector_skip_excluded/main_test.go

```diff
@@ -40,13 +40,15 @@ func TestMain(m *testing.M) {
 		util.HelmDeployLatestEraserRelease(util.TestNamespace,
 			"--set", util.ScannerEnable.Set("false"),
 			"--set", util.CollectorEnable.Set("false"),
+			"--set", util.RemoverImageRepo.Set(removerImage.Repo),
+			"--set", util.RemoverImageTag.Set(removerImage.Tag),
```

Comment:
Set the eraser image instead of the remover image, since the old chart uses v1alpha1.